src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseContentAccordionSection.tsx

```diff
@@ -123,100 +144,127 @@ const ReleaseContentAccordionSection = ({
   );
 
   return (
-    <EditableAccordionSection
-      {...props}
-      heading={heading}
-      disabledRemoveSectionTooltip={
-        hasLockedBlocks
-          ? 'This section is being edited and cannot be removed'
-          : undefined
-      }
-      caption={caption}
-      onHeadingChange={handleHeadingChange}
-      onRemoveSection={handleRemoveSection}
-      headerButtons={
-        <Tooltip
-          text="This section is being edited and cannot be reordered"
-          enabled={hasLockedBlocks}
-        >
-          {({ ref }) => (
-            <Button
-              ariaDisabled={hasLockedBlocks}
-              ref={ref}
-              variant={!isReordering ? 'secondary' : undefined}
-              onClick={async () => {
-                if (isReordering) {
-                  await reorderBlocks();
-                  setIsReordering(false);
-                } else {
-                  setIsReordering(true);
-                }
-              }}
-            >
-              {isReordering ? 'Save section order' : 'Reorder this section'}
-            </Button>
-          )}
-        </Tooltip>
-      }
-    >
-      {({ open }) => (
-        <>
-          <EditableSectionBlocks
-            blocks={blocks}
-            isReordering={isReordering}
-            sectionId={sectionId}
-            onBlocksChange={setBlocks}
-            renderBlock={block => (
-              <ReleaseBlock
-                block={block}
-                releaseId={release.id}
-                visible={open}
-              />
-            )}
-            renderEditableBlock={block => (
-              <ReleaseEditableBlock
-                allowComments
-                allowImages
-                block={block}
-                sectionId={sectionId}
-                sectionKey="content"
-                editable={!isReordering}
-                publicationId={release.publication.id}
-                releaseId={release.id}
-                visible={open}
-              />
+    <>
```

Comment:
Rather than having the modal at the root of the component, would just put it inside of the `EditableAccordionSection` children. There shouldn't be a difference and we don't end up creating a bunch of formatting related changes in this diff.